src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/LanguageVersionExtensions.cs

```diff
@@ -12,6 +12,9 @@ public static bool IsCSharp11OrAbove(this LanguageVersion languageVersion)
         public static bool HasConstantInterpolatedStrings(this LanguageVersion languageVersion)
             => languageVersion >= LanguageVersion.CSharp10;
 
+        public static bool IsCSharp12OrAbove(this LanguageVersion languageVersion)
+            => languageVersion >= LanguageVersion.Preview;
```

Comment:
Maybe some `TODO` here for replacing this with actual lang version 12 when it is actually arrives?